src/app/modules/design-patterns/observer/components/roulette-bet/roulette-bet.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { filter } from 'rxjs/operators';
+
+import { RoulettetBetModel } from '../../models/roulette-bet.model';
+import { RouletteService } from '../../services/roulette.service';
+
+
+@Component({
+  selector: 'app-roulette-bet',
+  templateUrl: './roulette-bet.component.html',
+  styleUrls: ['./roulette-bet.component.scss']
+})
+export class RouletteBetComponent implements OnInit, OnDestroy {
+  @Input() color: string;
+
+  betListArr: RoulettetBetModel[];
```

Comment:
sort class props

Reply:
change to bets

Reply:
Inputs come before or after public properties?

Reply:
Input is just a decorator added into varialbles, so, sort by variable name, ignore decorator